Sources/Login/LoginCoordinator.swift

```diff
@@ -128,6 +132,7 @@ extension LoginCoordinator: ParentCoordinator {
         case _ as OnboardingCoordinator:
             return
         case let child as AuthenticationCoordinator where child.loginError != nil:
+            introViewController?.enableIntroButton()
```

Comment:
When the user hits cancel on the authentication modal, `AuthenticationCoordinator.handleUniversalLink(_ URL:)` isn't called but an error is logged and the coordinator finishes so we need to ensure the button is reenabled at this point.
This method will be called twice when the `AuthenticationCoordinator` finishes with an error after `handleUniversalLink(_ URL:)` is called but that should be fine.